.github/workflows/ci-matrix-build.yml

```
@@ -0,0 +1,167 @@
+    if: ${{ always() && !cancelled() }}
+    needs:
+      - build-workflow
+      - dispatch-groups-linux-two-stage
+      - dispatch-groups-windows-two-stage
+      - dispatch-groups-linux-standalone
+      - dispatch-groups-windows-standalone
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+    outputs:
+      failure_message: ${{ steps.check-workflow.outputs.failure_message }}
```

actionlint (1.7.12)

[error] property "failure_message" is not defined in object type {success: string}

(expression)